dev-libs/quazip/quazip-0.7.6.recipe

```diff
@@ -56,7 +57,7 @@ BUILD()
 
 	cmake .. \
 		$cmakeDirArgs \
-		-DCMAKE_ROOT=$libDir/cmake
+		-DINSTALL_PATH=$libDir/cmake
```

Comment:
This works for quazip_x86:
```diff
-		-DINSTALL_PATH=$libDir/cmake
+		-DINSTALL_PATH=$prefix/lib/cmake
```

Reply:
Ìn INSTALL() you should add the next lines to move the library to the correct place:
```
	# move the libraries to the correct location
	if [ -n "$secondaryArchSuffix" ]; then
		mv $prefix/lib $prefix/lib2
		mkdir -p $libDir
		mv $prefix/lib2/* $libDir/
		rmdir $prefix/lib2
	fi
```

Reply:
Checked and working ok with the suggested changes :) (also for x86_64)